src/main/kotlin/io/sip3/salto/ce/Attributes.kt

```diff
@@ -33,6 +33,7 @@ object Attributes {
     const val error_type = "error_type"
     const val retransmits = "retransmits"
     const val duration = "duration"
+    const val distribution = "distribution"
```

Comment:
`distribution` is very common name i think. Or we change it in future if necessary?